pandas/tests/reshape/test_cut.py

```diff
@@ -585,3 +585,17 @@ def test_timedelta_cut_roundtrip():
         ["0 days 23:57:07.200000", "2 days 00:00:00", "3 days 00:00:00"]
     )
     tm.assert_index_equal(result_bins, expected_bins)
+
+
+@pytest.mark.parametrize("bins", [6, 7])
+@pytest.mark.parametrize(
+    "box, compare",
+    [(Series, tm.assert_series_equal), (np.array, tm.assert_categorical_equal)],
```

Comment:
Can you add `list` as a `box` since this should also work for something like `[True, True, False]`?